### ISSUE TITLE
ONNX Exporter to support multidim operators

### ISSUE THREAD

**Issue by dashesy** [pytorch/pytorch#20942]
## Motivation

I used something like `torch.mean(x, dim=(1,2,3))` for reduction. 
I get this in the export:

```python
/usr/local/lib/python2.7/dist-packages/torch/onnx/symbolic.py in symbolic(g, self, dim, keepdim)
    332         else:
    333             # dim-reduce path
--> 334             dim, keepdim = _get_const(dim, 'i', 'dim'), _get_const(keepdim, 'i', 'keepdim')
    335             return g.op(onnx_op_name, self, axes_i=[dim], keepdims_i=keepdim)

```

[multidim operators](https://github.com/pytorch/pytorch/issues/9703) for `mean` and `sum` are available since 1.0, and for `std`/`var` since 1.1

## Pitch

Support having `dim=(1,2,3)` for `torch.mean`, `torch.std`, ...

## Alternatives

I use this for alternative: `x.view(n_b, -1).mean(1).view(n_b, 1, 1, 1)`
## Additional context

Initially reported the issue [here](https://github.com/microsoft/onnxruntime/issues/1084)
Added [this issue](https://github.com/onnx/onnx/issues/2045) for ONNX to support `std`/`var`, but still `mean`  with multidim also should be supported

**Comment by neginraoof** [comment]
This is supported. We can close the issue.
